lib/signers/v4.js

```diff
@@ -11,10 +11,12 @@ var expiresHeader = 'presigned-expires';
  * @api private
  */
 AWS.Signers.V4 = inherit(AWS.Signers.RequestSigner, {
-  constructor: function V4(request, serviceName, signatureCache) {
+  constructor: function V4(request, serviceName, options) {
     AWS.Signers.RequestSigner.call(this, request);
     this.serviceName = serviceName;
-    this.signatureCache = signatureCache;
+    options = options || {};
+    this.signatureCache = options.signatureCache || true;
```

Comment:
Err, is there any way to turn off the signature cache? This will convert `false` to `true`. What about `this.signatureCache = options.signatureCache !== false;`